### PR TITLE
ci: update release please

### PR DESCRIPTION
**Pull request by kentakozuka**
Fixes https://github.com/bucketeer-io/bucketeer/issues/982

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+	"bump-minor-pre-major": true,
+	"bump-patch-for-minor-pre-major": true,
+	"draft": true,
```
Comment (cre8ivejp) [inline release-please-config.json:4]: What does this option?
Is it for the pull request or release?

Comment (kentakozuka) [reply release-please-config.json:4]: I thought it was for PR but apearantly, it was for release. I will make this false.

File: release-please-config.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	"prerelease": false,
+	"changelog-sections": [
+		{"type":"build","section":"Build System","hidden":false},
+		{"type":"chore","section":"Miscellaneous","hidden":false},
+		{"type":"feat","section":"Features","hidden":false},
+		{"type":"fix","section":"Bug Fixes","hidden":false},
+		{"type":"perf","section":"Performance Improvements","hidden":false},
+		{"type":"revert","section":"Reverts","hidden":false}
```
[the remark this answers is not in the document]
Comment (kentakozuka) [reply release-please-config.json]: 👍 
Screenshot 2024-06-04 at 16 35 33

**Comment by kentakozuka** [comment]
@cre8ivejp Fixed 👍

File: release-please-config.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (cre8ivejp) [reply release-please-config.json]: Nice!
Thanks!

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**